ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -19,8 +19,27 @@ const PredictionValue = styled.span`
 
 const NUM_TO_SHOW_ABOVE_THE_FOLD = 6 // how many predictors to show immediately
 
+const comparePathScores = (value, i, thresholds) => {
+  if (i < 2) { // Benign thresholds
+    if (i === 0) {
+      return value <= thresholds[0]
+    }
+    return (thresholds[0] === undefined || value > thresholds[0]) && value <= thresholds[1]
+  }
+
+  if (i === 2) { // Grey area
+    return (thresholds[1] === undefined || value > thresholds[1]) && value < thresholds[2]
+  }
+
+  // Pathogenic thresholds
+  if (i === 5) {
+    return true
+  }
+  return value >= thresholds[i - 1] && (thresholds[i] === undefined || value < thresholds[i])
```

Comment:
I changed the null thresholds to `undefined` because any numbers compared with `undefined` get a `false`. The null is similar to zero. A number compared with null can get `true` or `false` depending on whether it is negative or positive.

We still have to handle the benign and pathogen threshold differently because the sides of the close/open ends differ.